docs/content/anonymization/core-anonymizers.md

```diff
@@ -1,5 +1,11 @@
 # Core Anonymizers
 
+:::info
+All anonymizers can be configured via attributes on Doctrine ORM entities,
+but inheritance is not fully supported yet,
+[please read this page](doctrine-inheritance) for more information.
```

Comment:
This way is better supported by vitepress:

```suggestion
[please read this page](../doctrine-inheritance) for more information.
```

Reply:
And may be this block should go to `docs/content/anonymization/essentials.md`

Reply:
The file is in the same folder as the other one, so it seems that my URL is correct.